Gemfile.lock

```diff
@@ -0,0 +1,330 @@
+GEM
```

Comment:
@sakura-lxver can you add `Gemfile.lock` to `.gitignore`?

Reply:
ye